woocommerce/woocommerce-ios
# Order Creation: Add M1 Analytics Events

Part of #5717


# Description

This PR adds the analytics defined for the first Order Creation milestone.

- https://github.com/Automattic/tracks-events-registration/pull/719
- https://github.com/Automattic/tracks-events-registration/pull/720
- https://github.com/Automattic/tracks-events-registration/pull/721
- https://github.com/Automattic/tracks-events-registration/pull/722
- https://github.com/Automattic/tracks-events-registration/pull/723
- https://github.com/Automattic/tracks-events-registration/pull/724
- https://github.com/Automattic/tracks-events-registration/pull/725


# Testing instructions

- Launch the order creation flow and see the appropriate event being tracked
- Change order status and see the appropriate event being tracked
- Add a product and see the appropriate event being tracked
- Add customer details and see the appropriate event being tracked
- Tap the create button and see the appropriate event being tracked
- After success or failure see the appropriate event being tracked

---
